crates/re_types/src/components/scalar_ext.rs

```diff
@@ -0,0 +1,7 @@
+use super::Scalar;
+
+impl std::fmt::Display for Scalar {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{:.prec$}", self.0, prec = crate::DISPLAY_PRECISION)
```

Comment:
is it wise to have a hard-coded precision like this? we can give the control to the user instead with 

```suggestion
        self.0.fmt(f)
```

Reply:
I think you're right, now pondering if I should do the same to radius where I copy pasted this from :/